spec/integration/querying_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "manageiq_helper"
+
+RSpec.describe "Querying" do
+  as_user(:name => "Alice", :userid => "Alice", :password => "alicepassword") do
+    example "queries can be sent in the query string via the GET method" do
+      get(
+        "/graphql",
+        :headers => {"Authorization" => encode_basic_auth_credentials(user.userid, user.password)},
+        :params  => {:query => "{ viewer { name } }"}
+      )
+
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body).to eq("data" => {"viewer" => {"name" => "Alice"}})
+    end
+
+    example "queries can be sent in the query string via the POST method" do
+      post(
+        CGI.escape("/graphql?query={viewer{name}}"),
```

Comment:
Interesting to note: http://graphql.org/learn/serving-over-http/#get-request provides the follow an an example:

```
http://myapi/graphql?query={me{name}}
```

but according to RFC2396 this is not a valid URI! The curlies have to be escaped.